policybot/pkg/storage/spanner/read.go

```diff
@@ -239,3 +241,41 @@ func (s *store) ReadBotActivityByID(orgID string, repoID string) (*storage.BotAc
 
 	return &result, nil
 }
+
+func (s *store) ReadTestFlakyIssues(inactiveDays, createdDays int) ([]*storage.Issue, error) {
```

Comment:
Mind putting this in query.go instead?

The idea is read.go has simple "read just one thing" functions, whereas query uses query semantics that return N results.

Reply:
done.